cmd/lncli/commands.go

```diff
@@ -2913,11 +2913,40 @@ func describeGraph(ctx *cli.Context) error {
 var listPaymentsCommand = cli.Command{
 	Name:     "listpayments",
 	Category: "Payments",
-	Usage:    "List all outgoing payments.",
+	Usage:    "Query the payment history.",
+	Description: "This command enables the retrieval of payments stored in the " +
+		"database. Pagination is supported by the usage of " +
+		"index_offset in combination with the reversed flag. " +
+		"Pagination can be resumed by using the returned " +
+		"last_index_offset (for normal order), or first_index_offset " +
```

Comment:
nit: I'm not a fan of using the term normal to describe behaviour, it is somehow too descriptive and too general at the same time. Rather stick to forwards and reverse? Or a better word for forwards that I can't think of 🤔

Reply:
Agree, have used forwards instead of normal.